spec/features/tasks/use_tasks_test.go

```diff
@@ -8,11 +8,10 @@ import (
 
 func TestUseClusterTasks(t *testing.T) {
 	Convey("Given that Operator is installed", t, func() {
-
 		Convey("When I am logged in as a non-admin user", func() {
+			Convey("I Create a new namespace", nil)
```

Comment:
why is this step even needed?

Reply:
Listing and creating pipelines are 2 differnet tests

Reply:
makes sense created 2 tests